tests/test_router_bgp.rb

```diff
@@ -43,12 +52,14 @@ def create_bgp_vrf(asnum, vrf)
 end
 
 def setup_default
+  @device.cmd('no router bgp')
```

Comment:
Actually I shouldn't need it at all - because:

  def setup
    # Disable feature bgp before each test to ensure we
    # are starting with a clean slate for each test.
    super
    if platform == :ios_xr
      config('no router bgp')
    else
      config('no feature bgp')
    end
  end

Will retest to confirm.